modules/apps/batch-planner/batch-planner-web/src/main/resources/META-INF/resources/js/constants.js

```diff
@@ -55,6 +55,9 @@ export const PROCESS_COMPLETED = 'COMPLETED';
 export const PROCESS_FAILED = 'FAILED';
 export const PROCESS_STARTED = 'STARTED';
 
+
+export const OBJECT_DEFINITION =
```

Comment:
I prefer to use `OBJECT_DEFINITION_ENTITY_TYPE`, because you are using the entity type name instead of the instance of an Object Definition. What do you think?

Reply:
As well as for future PRs, I will add this change in an independent commit because this affects in several previous commits